app/src/main/java/com/example/placefinder/RetrofitClientInstance.kt

```diff
@@ -4,7 +4,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "foo"
+    private val BASE_URL = "https://data.ny.gov/"
```

Comment:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.